volatility3/framework/plugins/linux/tracing/tracepoints.py

```diff
@@ -258,11 +270,11 @@ def _generator(self):
         kernel_layer = self.context.layers[kernel.layer_name]
 
         if not kernel.has_symbol("__start___tracepoints_ptrs"):
-            raise exceptions.SymbolError(
-                "__start___tracepoints_ptrs",
-                self.vmlinux.symbol_table_name,
-                'The provided symbol table does not include the "__start___tracepoints_ptrs" symbol. This means you are either analyzing an unsupported kernel version or that your symbol table is corrupted.',
+            vollog.error(
+                'The provided symbol table does not include the "__start___tracepoints_ptrs" symbol.'
+                "This means you are either analyzing an unsupported kernel version or that your symbol table is corrupted."
             )
+            return
```

Comment:
Should we not still raise a symbol error, or do you want to continue after this?

Reply:
This is in the generator, so we want to report hard errors to the user. The symbol errors being raised has been for when plugins can into APIs. Raising symbol error in generator causes a huge backtrace instead of an orderly error message.

Reply:
Generally the way to do that is to catch the exception in the caller and report it then?  This way the caller has to do a check for a `None` response?  I'm kinda happy either way, but strictly the python way should be to throw the exception and catch it (so that other callers know something's really gone wrong, rather than just being able to sweep it under the rug)...

Reply:
This is in the generator though, so there is nothing to catch it, which is why we get the huge backtrace if we raise a symbolerror.

Reply:
Oh, I see, sorry, sometimes it's hard to spot what's what in the github UI.